travis.sh

```diff
@@ -4,6 +4,9 @@ set -e
 export GXX=g++-9
 export CC=gcc-9
 
+# Add the current working directory to $PATH so that tests can find bedrock.
```

Comment:
Why did this become necessary now?

Reply:
I took out the functions that looked for a `bedrock` binary by adding `../` to the front of the path a bunch of times, which got even more annoying in auth and the other repos. Now we just add it to the path for the tests and it works everywhere.